iarc_vision/scripts/apriltag_rebroadcast.py

```diff
@@ -1,4 +1,81 @@
 #!/usr/bin/env python2
 
+import rospy
+
+from std_msgs.msg import Header
+from apriltags_ros.msg import AprilTagDetectionArray, AprilTagDetection
+from geometry_msgs.msg import PoseWithCovarianceStamped, PoseWithCovariance, PointStamped, Point
+
+from iarc_main.msg import RoombaSighting, Roomba
+
+
+class TagTransformer(object):
+    """
+    A TagTransformer is responsible for transforming detections by the apriltags_ros library
+    into RoombaSighting messages for processing by the rest of the stack.
+    """
+    def __init__(self, linear_covariance=0.1, angular_covariance=0.5, tf_frame='map', camera_fov=1.0):
+        """
+        :param linear_covariance: (meters) the linear covariance associated with a tag detection
```

Comment:
might be nice to specify that these covariance values are arbitrary things we set in some comment.